src/components/overlays/EditPortalRoles/index.tsx

```diff
@@ -112,6 +114,14 @@ export default function EditPortalRoles({ id }: { id: string }) {
       assignedRoles.length === selectedRoles.length &&
       assignedRoles.every((value) => selectedRoles.includes(value)))
 
+  const disabledCheckbox = (currentRole: AppRole) => {
+    const allAdminRoles = allRoles.filter((item) => item.role.includes('Admin'))
+
+    return UserService.getUsername() === id
+      ? allAdminRoles.indexOf(currentRole) !== -1
```

Comment:
better use `a.includes(x)` instead of `a.indexOf(x) !== -1`

Reply:
Updated